packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -280,17 +290,36 @@ export class LionInputDatepicker extends ScopedElementsMixin(OverlayMixin(LionIn
     `;
   }
 
+  _setupOverlayCtrl() {
+    super._setupOverlayCtrl();
+
+    this.__datepickerBeforeShow = () => {
+      this._overlayCtrl.updateConfig(this._defineOverlayConfig());
+    };
+    this._overlayCtrl.addEventListener('before-show', this.__datepickerBeforeShow);
+  }
+
   /**
    * @override Configures OverlayMixin
    * @desc overrides default configuration options for this component
    * @returns {Object}
    */
   // eslint-disable-next-line class-methods-use-this
   _defineOverlayConfig() {
-    return {
-      ...withModalDialogConfig(),
-      hidesOnOutsideClick: true,
-    };
+    if (window.innerWidth >= 600) {
```

Comment:
Should we define a hardcoded breakpoint value on the lion layer?

Reply:
probable good enough for now... if someone needs it configurable then a feature request with some discussion will need to happen... (as we currently have no central breakpoints defined anywhere in lion)

and that would definitely be a separate pull request